### ISSUE TITLE
Implement EnvironmentMetrics 

### ISSUE THREAD

**Issue by gtmills** [ibm-openbmc/dev#3045]
https://redfish.dmtf.org/schemas/v1/EnvironmentMetrics.v1_0_0.json

Need EnvironmentMetrics  for PowerConsumedWatts
https://github.com/search?q=org%3Aopenbmc+PowerConsumedWatts&type=code

PowerConsumedWatts is called PowerWatts in EnvironmentMetrics

**Comment by zhanghaodi** [comment]
review by:
https://gerrit.openbmc-project.xyz/c/openbmc/bmcweb/+/43170

**Comment by gtmills** [comment]
Sorry for not responding sooner, at this time we just need PowerWatts in EnvironmentMetrics.
That is what we are trying to replace. 
I am fine if we also want to add Fan speeds but let's make sure we concentrate on PowerWatts which is the replacement for PowerConsumedWatts

**Comment by zhanghaodi** [comment]
@gtmills Okay, for PowerWatts in EnvironmentMetrics.
The data comes from /redfish/v1/Chassis/chassisId/Sensors/TotalPower;
For the sensor called TotalPower, it does not exist on our machine; 
this should be about reading the power of the entire chassis; how to get this data.
Does the sensor module need to add a sensor named TotalPower first?

**Comment by gtmills** [comment]
>  The data comes from /redfish/v1/Chassis/chassisId/Sensors/TotalPower;

Yes, https://github.com/openbmc/bmcweb/blob/9a128eb3fc3008f802974360e98de6495ed1a66b/redfish-core/lib/sensors.hpp#L1007
https://github.com/search?q=org%3Aopenbmc+total_power&type=code

It only exists on ours when the system is booted. 

>  Does the sensor module need to add a sensor named TotalPower first?

Anyway you can a sensor like this to your machine?

**Comment by zhanghaodi** [comment]
Thank you, I am trying to make a similar sensor exist in our machine.

**Comment by gtmills** [comment]
@zhanghaodi Can we pull this one and https://github.com/ibm-openbmc/dev/issues/2793 in downstream?

**Comment by mzipse** [comment]
refresh

**Comment by mzipse** [comment]
refresh again

**Comment by gtmills** [comment]
Still missing https://gerrit.openbmc.org/q/topic:redfish-EnvironmentMetrics
